groups/views.py

```diff
@@ -109,8 +115,23 @@ def form_valid(self, form):
             user=user,
         )
         self.pk = discussion.pk
+        self.email_subscribers(discussion)
         return super(DiscussionCreate, self).form_valid(form)
 
+    def email_subscribers(self, discussion):
+        """Notify all subscribers to the discussion's parent group, except its creator."""
+        users = discussion.group.watchers.exclude(pk=self.request.user.pk)
+        for user in users:
+            send(
+                to=user.email,
+                subject=NEW_DISCUSSION_SUBJECT.format(group=discussion.group.name),
+                template_name='groups/emails/new_discussion.txt',
+                context={
+                    'discussion': discussion,
+                    'user': user,
+                }
+            )
```

Comment:
It could, but then you'd have to put the `users=` logic into the `form_valid` method, and I wanted to put all the emailing code somewhere separate.  I don't think that's worth breaking for some test tweaks.